pkgs/applications/audio/open-stage-control/update.sh

```diff
@@ -8,18 +8,20 @@ nixpkgs="$(git rev-parse --show-toplevel || (printf 'Could not find root of nixp
 # Get latest release tag
 tag="$(curl -s https://api.github.com/repos/jean-emmanuel/open-stage-control/releases/latest | jq -r .tag_name)"
 
-# Download package.json from the latest release
-curl -sSL https://raw.githubusercontent.com/jean-emmanuel/open-stage-control/"$tag"/package.json | grep -v '"electron"\|"electron-installer-debian"' >"$nixpkgs"/pkgs/applications/audio/open-stage-control/package.json
+# Update hash
+updated="$(cd "$nixpkgs" && update-source-version "${UPDATE_NIX_ATTR_PATH:-open-stage-control}" "${tag#v}" --print-changes | jq -r length)"
 
-# Lock dependencies with node2nix
-node2nix \
-  --node-env "$nixpkgs"/pkgs/development/node-packages/node-env.nix \
-  --nodejs-16 \
-  --input "$nixpkgs"/pkgs/applications/audio/open-stage-control/package.json \
-  --output "$nixpkgs"/pkgs/applications/audio/open-stage-control/node-packages.nix \
-  --composition "$nixpkgs"/pkgs/applications/audio/open-stage-control/node-composition.nix
+if [ "$updated" -gt 0 ]; then
```

Comment:
```suggestion
if [[ -v updated ]]; then
  exit
fi
```

Reply:
Thank you for the review!!

This and the previous suggested change do not actually seem to work (notice that it always goes to the `if` branch for if the source version was updated):

```
[I] lily@bina ~/s/nixpkgs (master)> nix-shell -p common-updater-scripts --run 'updated="$(update-source-version open-stage-control 1.19.1 --print-changes)"; if [[ -v "$updated" ]]; then echo "*** not updated"; else echo "*** updated"; fi; echo "$updated"'
*** updated
[{"attrPath":"open-stage-control","oldVersion":"1.18.3","newVersion":"1.19.1","files":["/home/lily/src/nixpkgs/pkgs/applications/audio/open-stage-control/default.nix"]}]
[I] lily@bina ~/s/nixpkgs (master)> nix-shell -p common-updater-scripts --run 'updated="$(update-source-version open-stage-control 1.19.1 --print-changes)"; if [[ -v "$updated" ]]; then echo "*** not updated"; else echo "*** updated"; fi; echo "$updated"'
update-source-version: New version same as old version, nothing to do.
*** updated
[]
```

Reply:
Oh I see the confusion (just saw message on the review itself). `[ "$updated" -gt 0 ]` is relying on the number of packages updated, not the string length